src/popup/components/NetworkButton.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <BtnPlain
```

Comment:
There is already a button that looks like this: `BtnPill`. In my opinion it can be reused instead of creating additional CSS styling. Especially as we try to maintain a way of having common styles for all buttons.